samples/spring-boot-with-yaml/pom.xml

```diff
@@ -78,6 +79,17 @@
             </labels>
           </resources>
 
+          <generator>
```

Comment:
why not add a separate sample for JIB demonstration?

Reply:
Maybe we can make a seperate execution profile for Jib demonstration. A seperate sample isn't really required imo. @theexplorist make note.